client/src/test/scala/org/apache/toree/kernel/protocol/v5/client/socket/IOPubClientSpec.scala

```diff
@@ -108,7 +110,7 @@ class IOPubClientSpec extends TestKit(ActorSystem(
           .build
 
         // Mark as target being provided
-        doReturn(Some(mockCommCallbacks)).when(spyCommStorage)
+        doReturn(Some(mockCommCallbacks), Nil: _*).when(spyCommStorage)
```

Comment:
This is a Scala bug, fixed in 2.13.0. A workaround is required to make it happy with Scala 2.12.

https://github.com/scala/scala/pull/7680